### PR TITLE
Creating make file

### PR DESCRIPTION
**Pull request by wojtiwojt**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+create_venv:
```
Comment (spinus) [inline Makefile:1]: that's not good name anymore, probably better something like `docs`

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+[tool.poetry]
+name = "poetry-demo"
```
Comment (spinus) [inline pyproject.toml:2]: not a demo anymore :)

File: Makefile (hunk 1 above)
Comment (wojtiwojt) [reply Makefile]: done

File: pyproject.toml (hunk 2 above)
Comment (wojtiwojt) [reply pyproject.toml]: done

**Pull request merged**